plugins/obs-qsv11/common_utils_windows.cpp

```diff
@@ -9,23 +9,53 @@
 #include "common_directx9.h"
 #endif
 
+#define INTEL_VENDOR_ID 0x8086;
+
 /* =======================================================
  * Windows implementation of OS-specific utility functions
  */
 
-mfxStatus Initialize(mfxIMPL impl, mfxVersion ver, MFXVideoSession *pSession,
+mfxStatus Initialize(mfxVersion ver, mfxSession *pSession,
 		     mfxFrameAllocator *pmfxAllocator, mfxHDL *deviceHandle,
 		     bool bCreateSharedHandles, bool dx9hack)
 {
-	bCreateSharedHandles; // (Lain) Currently unused
-	pmfxAllocator;        // (Lain) Currently unused
+	bCreateSharedHandles; // (Hugh) Currently unused
```

Comment:
This looks like a bad conflict resolution, the correct name is Lain.

Reply:
My apologies, should be fixed now.